R/FilterState.R

```diff
@@ -449,11 +448,10 @@ FilterState <- R6::R6Class( # nolint
     # Set whether to keep NAs.
     #
     # @param value (`logical(1)`)
-    #   value(s) which come from the filter selection. Value is set in `server`
-    #   modules after selecting check-box-input in the shiny interface. Values are set to
-    #   `private$teal_slice$keep_na`
+    #   corresponding to the state of a checkbox input in the `shiny` interface.
     #
-    # @return NULL invisibly
+    # @return
+    # Sets `value` in `private$teal_slice$keep_na` and returns `NULL`, invisibly.
```

Comment:
(not a strong opinion)
I get the information about setting from the title. `@return` should describe what's returned so it's enough to describe just null invisibly

Reply:
Oooh, I made several similar changes, I must have missed this one.